core/server/src/main/java/alluxio/worker/AlluxioWorker.java

```diff
@@ -79,6 +80,9 @@ public static void main(String[] args) {
       System.exit(-1);
     }
 
+    // indicate the type of this process
+    AlluxioProcess.setType(AlluxioProcess.Type.WORKER);
```

Comment:
A question for @calvinjia : 
Here AlluxioWorker sets `AlluxioProcess.Type.WORKER`, when the decoupling is turned on, would the `worker client` be running as `WORKER` or `CLIENT`?